packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.slow-spec.ts

```diff
@@ -62,12 +62,32 @@ class ResultAccumulator implements SearchInWorkspaceClient {
 
 // Create a test file relative to rootDir.
 function createTestFile(filename: string, text: string) {
-    fs.writeFileSync(path.join(rootDir, filename), text);
+    const dir = getFilePathFromName(filename);
+    fs.writeFileSync(path.join(dir, filename), text);
     fileLines.set(filename, text.split('\n'));
 }
 
+const getFilePathFromName = (name: string) => {
```

Comment:
Can you document what this function does?

Would `getRootPathFromName` be more accurate?